nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -637,10 +638,12 @@ private boolean isCallToMethod(
       MethodInvocationNode invocationNode,
       Supplier<Type> containingTypeSupplier,
       String methodName) {
-    Symbol.MethodSymbol symbol = ASTHelpers.getSymbol(invocationNode.getTree());
+    MethodInvocationTree invocationTree = invocationNode.getTree();
+    Symbol.MethodSymbol symbol = ASTHelpers.getSymbol(invocationTree);
     return symbol != null
         && symbol.getSimpleName().contentEquals(methodName)
-        && ASTHelpers.isSubtype(symbol.owner.type, containingTypeSupplier.get(state), state);
+        && ASTHelpers.isSubtype(
+            ASTHelpers.getReceiverType(invocationTree), containingTypeSupplier.get(state), state);
```

Comment:
This is pretty subtle, do we want to add a comment here stating that `symbol.owner.type` won't work for certain compilers?